indicators/instagram-tpexkd.yml

```diff
@@ -0,0 +1,38 @@
+title: Instagram Phishing Kit TPEXkd
+description: |
+    Detects a phishing kit targeting Instagram.
+    Commonly deployed on Freenom domains.
+
+
+references:
+    - https://urlscan.io/result/b861f41c-57f1-445f-bf6d-d80d84f673b4/
+    - https://urlscan.io/result/b7fdc491-68bb-42ed-853b-a24623b3dc5b/
+    - https://urlscan.io/result/aa338e86-53b1-4cb5-8c15-aea917601246/
+    - https://urlscan.io/result/bb82b60a-3de6-4c71-896a-3349ebbcca4c/
+
+detection:
+
+    fontAwesome:
+      html|contains:
```

Comment:
This JS file could be referenced on benign websites please consider using a different indicator such as the class name of the button `yasinbaykam` as this seems to be shared across all the occurrences of this kit you have supplied.